test/common/index.js

```diff
@@ -0,0 +1,228 @@
+// Common tests that are run once for
+// hypercore, hyperdb, and hyperdrive.
+
+module.exports = function () {
+
+  var tape = require('tape')
+  var database = require('hyperdrive')
```

Comment:
Also, I’d forgotten this here and wasn’t actually passing the database in so I was testing hyperdrive three times (fun). Fixed that but tests still failing. Very odd. Something is fragile but I don’t know what. Going to keep looking.